doc/developers/contributing.rst

```diff
@@ -213,6 +213,22 @@ then submit a "pull request" (PR):
     and start making changes. Always use a ``feature`` branch. It's good practice to
     never work on the ``master`` branch!
 
+.. note::
+
+  In the above setup, your ``origin`` remote repository points to
+  ``YourLogin/scikit-learn.git``. If you wish to fetch/merge from the main
+  repository instead of your forked one, you will need to add another remote
+  to use instead of ``origin``. If we choose the name ``upstream`` for it, the
+  command will be::
+
+        $ git remote add upstream https://github.com/scikit-learn/scikit-learn.git
+
+  And in order to fetch the new remote and base your work on the latest changes
+  of it you can::
+
+        $ git fetch upstream
+        $ git checkout -b my-feature upstream/master
```

Comment:
Maybe it's worth just changing the instructions above rather than adding a note. But I'm fine with this